municipal_finance/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-25 11:05
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.contrib.sites.models import Site
+
+
+def create_sites(apps, schema_editor):
```

Comment:
did you write this manually or was this generated from something you did on the shell? I want to add municipal-finance.code4sa.org to either serve the API site or redirect to data.municipalmoney.org.za